### DISCUSSION TITLE
DPAT-9 'Deuce'

### DISCUSSION THREAD

**Discussion by Sol-au-fein** [secretcowlevel/DoomtrooperPublic#49] [Card Balance]
S +6. 
As 1 Tactical Action, you may destroy any SUPPORT

**Comment by TomekPL-1** [comment]
DPAT-9 'Deuce' (CAPITOL) [C]
EQUIPMENT, FIREARM
[S] +6. As 1 Tactical Action, you may discard any SUPPORT, if you spend in addition 1 action you may annihilate any SUPPORT.
